tool/gocross/gocross.go

```diff
@@ -15,9 +15,9 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"runtime/debug"
 
 	"tailscale.com/atomicfile"
-	"tailscale.com/version"
```

Comment:
maybe lock this in with a new deptest.DepChecker test? they're easy to add if so.